portfolio/src/main/webapp/script.js

```diff
@@ -71,6 +71,24 @@ function contact() {
 	console.log('Contact!'); 
 }
 
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {
+          center: {lat: 37.422, lng: -122.084}, 
+          zoom: 16,
+          styles: [
+              {
+               'featureType': 'water',
```

Comment:
In the [api examples](https://developers.google.com/maps/documentation/javascript/styling) these map keys aren't quoted. Is there a reason they need to be?

Reply:
N/A anymore for this PR